en/restore-from-ebs-snapshot-across-multiple-kubernetes.md

```diff
@@ -0,0 +1,227 @@
+
+**You must execute the following steps in the control plane**.
+
+Depending on the authorization method you choose in the previous step for granting remote storage access, you can restore data to TiDB using any of the following methods accordingly:
+
+> **Note:**
+>
+> Snapshot restore creates volumes with the default configuration (3000 IOPS/125 MB/s) of GP3. To perform restore using other configurations, you can specify the volume type or configuration, such as `--volume-type=gp3`, `--volume-iops=7000`, or `--volume-throughput=400`, and they are shown in the following examples.
+
+<SimpleTab>
+<div label="AK/SK">
+
+If you grant permissions by accessKey and secretKey, you can create the `VolumeRestore` CR as follows:
+
+```shell
+kubectl apply -f restore-fed.yaml
+```
+
+The `restore-fed.yaml` file has the following content:
+
+```yaml
+---
+apiVersion: federation.pingcap.com/v1alpha1
+kind: VolumeRestore
+metadata:
+  name: ${restore-name}
+spec:
+  clusters:
+  - k8sClusterName: ${k8s-name1}
+    tcName: ${tc-name1}
+    tcNamespace: ${tc-namespace1}
+    backup:
+      s3:
+        provider: aws
+        secretName: s3-secret
+        region: ${region-name}
+        bucket: ${bucket-name}
+        prefix: ${backup-path1}
+  - k8sClusterName: ${k8s-name2}
+    tcName: ${tc-name2}
+    tcNamespace: ${tc-namespace2}
+    backup:
+      s3:
+        provider: aws
+        secretName: s3-secret
+        region: ${region-name}
+        bucket: ${bucket-name}
+        prefix: ${backup-path2}
+  - ... # other clusters
+  template:
+    br:
+      sendCredToTikv: true
+      options:
+      - --volume-type=gp3
+      - --volume-iops=7000
+      - --volume-throughput=400
+    toolImage: ${br-image}
+    warmup: sync
+    warmupImage: ${wamrup-image}
+```
+
+</div>
+
+<div label="IAM role with Pod">
+
+If you grant permissions by associating Pod with IAM, you can create the `VolumeRestore` CR as follows:
+
+```shell
+kubectl apply -f restore-fed.yaml
+```
+
+The `restore-fed.yaml` file has the following content:
+
+```yaml
+---
+apiVersion: federation.pingcap.com/v1alpha1
+kind: VolumeRestore
+metadata:
+  name: ${restore-name}
+  annotations:
+    iam.amazonaws.com/role: arn:aws:iam::123456789012:role/role-name
+spec:
+  clusters:
+  - k8sClusterName: ${k8s-name1}
+    tcName: ${tc-name1}
+    tcNamespace: ${tc-namespace1}
+    backup:
+      s3:
+        provider: aws
+        region: ${region-name}
+        bucket: ${bucket-name}
+        prefix: ${backup-path1}
+  - k8sClusterName: ${k8s-name2}
+    tcName: ${tc-name2}
+    tcNamespace: ${tc-namespace2}
+    backup:
+      s3:
+        provider: aws
+        region: ${region-name}
+        bucket: ${bucket-name}
+        prefix: ${backup-path2}
+  - ... # other clusters
+  template:
+    br:
+      sendCredToTikv: false
+      options:
+      - --volume-type=gp3
+      - --volume-iops=7000
+      - --volume-throughput=400
+    toolImage: ${br-image}
+    warmup: sync
```

Comment:
@WangLe1321 Please make sure recommendation of sync warmup is delivered in faq for restore performance.